libs/domain/picking/src/components/picking/picking.styles.ts

```diff
@@ -62,4 +62,8 @@ export const styles = css`
     font-size: 18px;
     font-weight: 600;
   }
+
+  oryx-tab[selected] oryx-chip {
+    background-color: var(--oryx-color-success-100);
```

Comment:
Assuming our latest discussion on design approach, I would assume that we should not do such style tweaks on the component level anymore?
cc @tobi-or-not-tobi

Reply:
We should stick to the system here, see https://test-storybooks.netlify.app/?path=/story/ui-graphical-chip-static--states. Any reason to not use the success type here?